_includes/integrations/react-quick-start.md

```diff
@@ -0,0 +1,135 @@
+
+    ```
+    $ npm install -g create-react-app
+    ```
+2. Create a new React project named `tinymce-react-demo`.
+
+    ```
+    $ create-react-app tinymce-react-demo
+    ```
+3. Change into the newly created directory.
+
+    ```
+    $ cd tinymce-react-demo
+    ```
+4. Install the `tinymce-react` package and save it to your `package.json` with `--save`.
+
+    ```
+    $ npm install --save @tinymce/tinymce-react
+    ```
+5. Using a text editor, open `/path/to/tinymce-react-demo/src/App.js` and replace the contents with:
+
+    ```js
+    import React from 'react';
+    import { Editor } from '@tinymce/tinymce-react';
+
+    class App extends React.Component {
+      handleEditorChange = (e) => {
+        console.log('Content was updated:', e.target.getContent());
+      }
+
+      render() {
+        return (
+          <Editor
+            initialValue="<p>This is the initial content of the editor</p>"
+            init={% raw %}{{{% endraw %}
+              height: 500,
+              menubar: false,
+              plugins: [
+                'advlist autolink lists link image charmap print preview anchor',
+                'searchreplace visualblocks code fullscreen',
+                'insertdatetime media table paste code help wordcount'
+              ],
+              toolbar:
+                'undo redo | formatselect | bold italic backcolor | \
+                alignleft aligncenter alignright alignjustify | \
+                bullist numlist outdent indent | removeformat | help'
+            {% raw %}}}{% endraw %}
+            onChange={this.handleEditorChange}
+          />
+        );
+      }
+    }
+
+    export default App;
+    ```
+    This JavaScript file will create the class `App` containing a TinyMCE editor configured to replicate the example on the [Basic example page]({{site.baseurl}}/docs/demo/basic-example/).
+6. Provide access to TinyMCE using Tiny Cloud, TinyMCE Self-hosted or by bundling TinyMCE using a module loader.
+
+    * **Tiny Cloud**
+
+        Include the `apiKey` option in the editor element and include your [TinyMCE API key]({{site.baseurl}}/signup/).
+
+        Such as:
+
+        ```js
+        <Editor apiKey='your-api-key' init={% raw %}{{{% endraw %} /* your other settings */ {% raw %}}}{% endraw %} />
+        ```
+
+    * **TinyMCE Self-hosted**
+
+        To load a self-hosted deployment of TinyMCE, add a script to either the `<head>` or the end of the `<body>` of the HTML file, such as:
+        ```html
+        <script src="/path/to/tinymce.min.js"></script>
+        ```
+
+        To use a self-hosted deployment of TinyMCE with this React application, add the script to `/path/to/tinymce-react-demo/public/index.html`.
+
+        For information on self-hosting TinyMCE, see: [Advanced installation choices]({{site.baseurl}}/general-configuration-guide/advanced-install/).
+
+    * **Bundling TinyMCE using a module loader**
```

Comment:
This is also self-hosted so maybe have this content under the "TinyMCE Self-hosted" section and present it as two alternatives?

Reply:
True. updating.